src/schemas/21-527/schema.js

```diff
@@ -7,10 +7,80 @@ let schema = {
   title: 'INCOME, NET WORTH, AND EMPLOYMENT STATEMENT',
   type: 'object',
   additionalProperties: false,
-  definitions: _.pick(definitions,
-    'dateRange',
-    'netWorthAccount'
-  ),
+  definitions: _.merge(_.pick(definitions,
+    'dateRange'
+  ), {
+    netWorth: {
+      type: 'object',
+      properties: {
+        bank: { type: 'integer' },
+        ira: { type: 'integer' },
+        stocks: { type: 'integer' },
+        business: { type: 'integer' },
+        realProperty: { type: 'integer' },
+        otherProperty: { type: 'integer' },
+        additionalSources: { $ref: '#/definitions/additionalSources' }
+      }
+    },
+    additionalSources: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          name: {
+            type: 'string'
+          },
+          amount: {
+            type: 'integer'
+          }
+        }
+      }
+    },
+    monthlyIncome: {
+      type: 'object',
+      properties: {
+        salary: {
```

Comment:
527EZ doesn't have salary but adds a field called "service retirement" which is separate from civil service